docs/src/reference/3_backend.md

```diff
@@ -0,0 +1,42 @@
+# Backend
```

Comment:
I would call the whole page "Utilities and underlying structures" instead of Backend

Reply:
Knock yourself out! Just beware that it fits in the side bar